### ISSUE TITLE
How to change default "open in browser as"

### ISSUE THREAD

**Issue by gabri25ele** [Rob--W/open-in-browser#48]
this is the default choice open in browser as:
![1annotazione](https://user-images.githubusercontent.com/33927879/47845568-acab3f80-ddc5-11e8-8c06-28431307c1bf.png)

How can I set TEXT as **default**, like this:
![2annotazione](https://user-images.githubusercontent.com/33927879/47845617-cc426800-ddc5-11e8-8fb3-5dcc8d632ea0.png)

**Comment by gabri25ele** [comment]
Everytime, I need to select it... I would use ever the text choice, to transfer the link in my download manager.
OR...
for example in a filecrypt link: https://filecrypt.cc/Container/BAE5194A17.html
I need to click on DLC and the select TEXT to generate the content:

> 9dlHdiru0mAaZ2hl6L5tT05YRjamVeutSeJixbo+hBRe2Mcf+b/yJn8nUkEPI72k3kv/CsV/B1UO7Hl+y5403IgDPZRR6bSVxmxU+3A+NljYBx1GhrvPGimVPx/gQSnpQIcdcF495UKJA3FjAFGVZbMRpIvKHzE7rEyA1XSYw2PdedqN3ss3NO/G5Iv2ubwquS6WqI5DzHDRNQ+sMLAQ7bt7EzNsFI8g0mJhwaIPYYqcGqxja5jgp5qOrSwk1zsvcmDjKTzRsasyTH1qNqtR0UsMEpbfcndOWx5c4P4VjKdlDf/lJUKLzaIqHproAIe1XpwLDMXKhwcXYwI2QLih+XzmZwwae5ZktFDzmnTC3uZ3BpExlfOBzZZPvnI6HAXItTABABkPYH4xbCdRN5S6mrcZQvOtS/PEuRInIVequi4ISf7fCfJPbqy6CqCYxD0/BuvoxE+FMXvsnKlvvybgFqz7UmOgzyCmWDdGHSVPi2hBEKrqtiw9xuuD99f/f0LpfVHHIWbLufWC0sE5k5CK8KBfYwSXzQDklb3IChHtaOm/ATh9LHWx3PFpRT1SLeD7w22pS67ShlQ53ZcxFLcQt2CQUeof2MlcYlLHN3b4wXN0PrS355pv7xTMlbuq+fJAgAHNKg/3dMoMOSxtfTp0ttp5yTvKc4X9UUKMGViaK+blJUK4NKA2/thJnSOWOqm0RPVrWwH7nYtV83zKz/tul5plHzwq8z4pqeShgpOLtjgLWC0iPwoSVje+qCbzYPPwFl1HnVrERbfx9BWL1xdE2MJ2GmtuyFbQWL7xw/zwRh2en3A9UUasb+G+s2QWf0Gjymh03zGF1yeyazkHN9Vs+WoAGEwRg77uK5S606P3GHkV1EGa1gWSU7/n6/lC+763r95z5cS3J9WUZF+s2bNObRd3DHWkc0dx+SObx0F9E7O7fk4F/lrDB5cc98Z9T5dUlAKxU1iftM5GIf5j2ZIXmok7+bC2dUu8XlV5ZyV1bwSBv7lV73P33aALRm2My4tM2XgHCm9i6DVgA88KnGsaklJQvSul3aAQaFVitzfuFVg2PEK1foCumEuNCyA6AgS3NP4nsqAPmHk7f7RUhhop++Erlfje8v+oBnF/2HM7GJEVf4ZbxH+TKk6yL9nR7cIXbLrvny1rl45dA5ok9PcXssUa5B3DJbnkH/+eDNflPdUdEsyBZ9cDN+5ouRS43zFMyiMux4qf3m0D4ZoMjAEEzWKm4HKOiTWTSY89UwWO0sb2FeKSQwDcTfhOuKVCUi8Mkdji6TLfuALkIR081qKGblvgGCcbosW8r6YyIlrT8CQ=bllqU3luR1hYZzVwR1NvMGtMNDlIQUtJWFhzK1cxbWlpUyswQmYweGpxNWNLV0ZqN2IzaElDTGZ5dTdTdWEvVA==

to paste directly here: http://dcrypt.it/

**Comment by Rob--W** [comment]
Sorry, I'm not completely following your comment. Are you saying that the (text) response usually contains content that you'd like to display in the browser? If so, have you tried the Open in browser as Text + selecting the checkbox?

If not, please clarify. Your request also sounds familiar; in #47 you've also requested a similar feature.

**Comment by gabri25ele** [comment]
yes,  I'm saying that the text response content I'd like to display in the browser.
This is more useful with dlc content (did you see the example?), and also for the link that don't pass automatically in my download manager (https://www.internetdownloadmanager.com/)
I use this extension in Particolar for that.

and YES also for the #47 (compromise)

**Comment by Rob--W** [comment]
Have you tried:

> If so, have you tried the Open in browser as Text + selecting the checkbox?

And for #47, have you tried what I suggested in a comment?

**Comment by gabri25ele** [comment]
with ctrl+a/c I have the selection of the window

![cattura2](https://user-images.githubusercontent.com/33927879/47848770-a752f280-ddcf-11e8-94a7-54bc92f1c9cd.PNG)

:-/

but are not we moving away from the thread?
we want ever the open in browser like a text
I honestly do not know what to do with the mime

If I select the MIME, Firefox give me a new save windows like this
![cattura3](https://user-images.githubusercontent.com/33927879/47848883-103a6a80-ddd0-11e8-9fd3-7737a95df425.PNG)

**Comment by Rob--W** [comment]
You can use the "Da ora in avanti esegui questa azione per tutti i file di questo tipo." option to always default to opening as text. Would this address your needs?

In your specific case, the "dlc" extension is not recognized, which means that the option applies to all downloads rather than the specific file extension, which is not good. I intend to look into a fix in a future update, so that the file extension can also be used if the type could not be determined.

PS. Following your information in https://github.com/Rob--W/open-in-browser/issues/48#issuecomment-434999523 , I found my requested information:

```
HTTP/1.1 200 OK
Content-Type: application/octet-stream
Content-Disposition: attachment; filename="c968f19751929403b8bf3a32da9c53576bcb811b.dlc"
```

This means that the only way to distinguish your "download" from other "downloads" is by the proposed file extension.
